src/nnue/layers/affine_transform.h

```diff
@@ -489,12 +496,14 @@ namespace Stockfish::Eval::NNUE::Layers {
 #if defined (USE_SSSE3)
       const auto inputVector = reinterpret_cast<const vec_t*>(input);
 
-      static_assert(OutputDimensions % OutputSimdWidth == 0 || OutputDimensions == 1);
+      constexpr std::size_t accumulatorElementCount = sizeof(vec_t) / sizeof(OutputType);
```

Comment:
I'd rather move OutputSimdWidth and InputSimdWidth definitions to here. Right now it makes them inconsistent.

Reply:
OK, I will try that. Should I also update the comment about AVX-512 at the beginning of the file?